2021-11-17 03:55:50,468 | 2855 | tracing_net.net, __init__, 99 | INFO | set up on-demand mininet
2021-11-17 03:55:50,469 | 2855 | tracing_of_pipeline, start, 123 | INFO | OpenFlow Tracer Start
2021-11-17 03:55:50,477 | 2855 | tracing_net.net, start, 312 | INFO | mininet start
2021-11-17 03:55:50,483 | 2855 | tracing_of_pipeline.analyzer, start_analyzing, 143 | INFO | start analyzing interval (start_time = 1637121350.482959, interval = 2)
2021-11-17 03:55:51,630 | 2855 | tracing_net.net, start_tracing, 526 | INFO | tracing start
2021-11-17 03:55:51,701 | 3024 | tracing_net.util.packet_capture, start_capture, 65 | INFO | packet capture start on s1-eth1
2021-11-17 03:55:51,719 | 3026 | tracing_net.util.packet_capture, start_capture, 65 | INFO | packet capture start on s1-eth2
2021-11-17 03:55:51,727 | 2855 | tracing_net.flowtable.flow_monitor, __init__, 46 | INFO | flow monitor start : s1
2021-11-17 03:55:51,735 | 3030 | tracing_net.util.packet_capture, start_capture, 65 | INFO | packet capture start on s1-eth3
2021-11-17 03:55:51,722 | 3028 | tracing_net.util.packet_capture, start_capture, 65 | INFO | packet capture start on s2-eth2
2021-11-17 03:55:51,749 | 2855 | tracing_net.flowtable.flow_monitor, __init__, 46 | INFO | flow monitor start : s2
2021-11-17 03:55:52,486 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:55:52,488 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:55:52,489 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:55:52,490 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:55:52,498 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-17 03:55:52,499 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-17 03:55:52,502 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-17 03:55:52,503 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-17 03:55:52,797 | 3033 | tracing_net.flowtable.flow_monitor, read_dump_flow, 83 | WARNING | No result dump flows
2021-11-17 03:55:52,837 | 3040 | tracing_net.flowtable.flow_monitor, read_dump_flow, 83 | WARNING | No result dump flows
2021-11-17 03:55:53,794 | 3033 | tracing_net.flowtable.flow_monitor, read_dump_flow, 83 | WARNING | No result dump flows
2021-11-17 03:55:53,849 | 3040 | tracing_net.flowtable.flow_monitor, read_dump_flow, 83 | WARNING | No result dump flows
2021-11-17 03:55:54,486 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:55:54,487 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:55:54,489 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:55:54,489 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:55:54,493 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-17 03:55:54,493 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-17 03:55:54,493 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-17 03:55:54,494 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-17 03:55:54,825 | 3033 | tracing_net.flowtable.flow_monitor, read_dump_flow, 83 | WARNING | No result dump flows
2021-11-17 03:55:54,839 | 3040 | tracing_net.flowtable.flow_monitor, read_dump_flow, 83 | WARNING | No result dump flows
2021-11-17 03:55:55,861 | 3033 | tracing_net.flowtable.flow_monitor, read_dump_flow, 83 | WARNING | No result dump flows
2021-11-17 03:55:55,866 | 3040 | tracing_net.flowtable.flow_monitor, read_dump_flow, 83 | WARNING | No result dump flows
2021-11-17 03:55:56,487 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:55:56,487 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:55:56,488 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:55:56,488 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:55:56,492 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-17 03:55:56,492 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-17 03:55:56,492 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-17 03:55:56,493 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-17 03:55:56,771 | 3033 | tracing_net.flowtable.flow_monitor, read_dump_flow, 83 | WARNING | No result dump flows
2021-11-17 03:55:56,839 | 3040 | tracing_net.flowtable.flow_monitor, read_dump_flow, 83 | WARNING | No result dump flows
2021-11-17 03:55:58,486 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:55:58,487 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:55:58,487 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:55:58,488 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:55:58,490 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-17 03:55:58,491 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-17 03:55:58,491 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-17 03:55:58,492 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-17 03:56:00,485 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:00,487 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:00,487 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:00,488 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:00,491 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-17 03:56:00,491 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-17 03:56:00,492 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-17 03:56:00,493 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-17 03:56:02,486 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:02,486 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:02,487 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:02,488 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:02,490 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-17 03:56:02,491 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-17 03:56:02,491 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-17 03:56:02,492 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-17 03:56:04,486 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:04,486 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:04,487 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:04,487 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:04,490 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-17 03:56:04,491 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-17 03:56:04,491 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-17 03:56:04,492 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-17 03:56:06,486 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:06,499 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:06,505 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:06,505 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:06,509 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-17 03:56:06,510 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-17 03:56:06,510 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-17 03:56:06,511 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-17 03:56:08,489 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:08,489 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:08,490 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:08,491 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-17 03:56:08,494 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-17 03:56:08,494 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-17 03:56:08,495 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-17 03:56:08,496 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-17 03:56:10,486 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [<Msg captured_interface=s1-eth1, sniff_timestamp=1637121369.593496, pkt=<ARP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>, <Msg captured_interface=s1-eth1, sniff_timestamp=1637121369.668054, pkt=<ARP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>, <Msg captured_interface=s1-eth1, sniff_timestamp=1637121369.6680696, pkt=<ICMP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>], 's1-eth3': [<Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.6024306, pkt=<ARP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>, <Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.6024745, pkt=<ARP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>, <Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.7038112, pkt=<ICMP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>]})
2021-11-17 03:56:10,487 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [<Msg captured_interface=s1-eth1, sniff_timestamp=1637121369.593496, pkt=<ARP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>, <Msg captured_interface=s1-eth1, sniff_timestamp=1637121369.668054, pkt=<ARP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>, <Msg captured_interface=s1-eth1, sniff_timestamp=1637121369.6680696, pkt=<ICMP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>], 's1-eth3': [<Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.6024306, pkt=<ARP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>, <Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.6024745, pkt=<ARP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>, <Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.7038112, pkt=<ICMP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>]})
2021-11-17 03:56:10,488 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-17 03:56:10,489 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-17 03:56:10,490 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-17 03:56:10,490 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-17 03:56:12,487 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [<Msg captured_interface=s1-eth1, sniff_timestamp=1637121369.593496, pkt=<ARP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>, <Msg captured_interface=s1-eth1, sniff_timestamp=1637121369.668054, pkt=<ARP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>, <Msg captured_interface=s1-eth1, sniff_timestamp=1637121369.6680696, pkt=<ICMP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>], 's1-eth3': [<Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.6024306, pkt=<ARP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>, <Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.6024745, pkt=<ARP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>, <Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.7038112, pkt=<ICMP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>]})
2021-11-17 03:56:12,488 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [<Msg captured_interface=s1-eth1, sniff_timestamp=1637121369.593496, pkt=<ARP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>, <Msg captured_interface=s1-eth1, sniff_timestamp=1637121369.668054, pkt=<ARP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>, <Msg captured_interface=s1-eth1, sniff_timestamp=1637121369.6680696, pkt=<ICMP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>], 's1-eth3': [<Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.6024306, pkt=<ARP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>, <Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.6024745, pkt=<ARP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>, <Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.7038112, pkt=<ICMP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>]})
2021-11-17 03:56:12,489 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-17 03:56:12,489 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-17 03:56:12,490 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-17 03:56:12,490 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-17 03:56:14,486 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's1-eth3': [<Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.6024306, pkt=<ARP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>, <Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.6024745, pkt=<ARP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>, <Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.7038112, pkt=<ICMP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>]})
2021-11-17 03:56:14,488 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's1-eth3': [<Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.6024306, pkt=<ARP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>, <Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.6024745, pkt=<ARP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>, <Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.7038112, pkt=<ICMP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>]})
2021-11-17 03:56:14,506 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-17 03:56:14,507 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-17 03:56:16,487 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [<Msg captured_interface=s1-eth1, sniff_timestamp=1637121374.7319727, pkt=<ARP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>], 's1-eth3': [<Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.7038426, pkt=<ICMP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>, <Msg captured_interface=s1-eth3, sniff_timestamp=1637121374.7317789, pkt=<ARP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>]})
2021-11-17 03:56:16,487 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [<Msg captured_interface=s1-eth1, sniff_timestamp=1637121374.7319727, pkt=<ARP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>], 's1-eth3': [<Msg captured_interface=s1-eth3, sniff_timestamp=1637121369.7038426, pkt=<ICMP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>, <Msg captured_interface=s1-eth3, sniff_timestamp=1637121374.7317789, pkt=<ARP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>]})
2021-11-17 03:56:16,493 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-17 03:56:16,494 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-17 03:56:18,486 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's1-eth3': [<Msg captured_interface=s1-eth3, sniff_timestamp=1637121374.7317789, pkt=<ARP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>]})
2021-11-17 03:56:18,487 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's1-eth3': [<Msg captured_interface=s1-eth3, sniff_timestamp=1637121374.7317789, pkt=<ARP_RAW Packet>, in_port=s1-eth3, in_phy_port=s1-eth3>]})
2021-11-17 03:56:18,496 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-17 03:56:18,496 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-17 03:56:20,486 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's1-eth3': []})
2021-11-17 03:56:20,487 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's1-eth3': []})
2021-11-17 03:56:20,495 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-17 03:56:20,495 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-17 03:56:22,486 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's1-eth3': []})
2021-11-17 03:56:22,487 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's1-eth3': []})
2021-11-17 03:56:22,488 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-17 03:56:22,489 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-17 03:56:22,489 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-17 03:56:22,490 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-17 03:56:23,289 | 2855 | tracing_net.net, stop_tracing, 542 | INFO | tracing stop
2021-11-17 03:56:23,291 | 2855 | tracing_of_pipeline.analyzer, stop_analyzing, 151 | INFO | stop analyzing interval. (last count = 16)
2021-11-17 03:56:23,302 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth3': []})
2021-11-17 03:56:23,307 | 2855 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth3': []})
2021-11-17 03:56:23,309 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-17 03:56:23,310 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-17 03:56:23,311 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-17 03:56:23,311 | 2855 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-17 03:56:23,312 | 2855 | tracing_of_pipeline.analyzer, analyze_rest, 163 | INFO | These packets could not be analyzed. {'l2': [], 'l1': [], 'l3': [], 'l4': []}
2021-11-17 03:56:23,312 | 2855 | tracing_of_pipeline.analyzer, analyze_rest, 164 | INFO | These OpenFlowMessage could not be analyzed. {'s1': [<src.ofproto.msg.MsgForOFMsg object at 0x7f8c5bc5e730>, <src.ofproto.msg.MsgForOFMsg object at 0x7f8c581e1280>, <src.ofproto.msg.MsgForOFMsg object at 0x7f8c581e12b0>, <src.ofproto.msg.MsgForOFMsg object at 0x7f8c581e1310>, <src.ofproto.msg.MsgForOFMsg object at 0x7f8c581e14c0>, <src.ofproto.msg.MsgForOFMsg object at 0x7f8c581e15b0>], 's2': [<src.ofproto.msg.MsgForOFMsg object at 0x7f8c581e15e0>, <src.ofproto.msg.MsgForOFMsg object at 0x7f8c581e14f0>]}
2021-11-17 03:56:23,398 | 2855 | tracing_of_pipeline, stop, 137 | INFO | OpenFlow Tracer Stop
2021-11-17 03:56:23,403 | 2855 | tracing_net.net, stop_tracing, 542 | INFO | tracing stop
2021-11-17 03:56:23,412 | 2855 | tracing_net.net, stop, 317 | INFO | mininet stop
